main_test.go

```diff
@@ -49,3 +49,20 @@ func TestFindRoot(t *testing.T) {
 		}
 	}
 }
+
+func TestSlashedGOPATH(t *testing.T) {
+	gopath := os.Getenv("GOPATH")
```

Comment:
Might be time to deal with @peterbourgon's [complaint TODO about `newContext()` being global](https://github.com/golang/dep/blob/1b193f4439655572d59fe1b87035870f1a7344ca/main.go#L72-L73) - that should let us inject what we want directly. Then we could rely on the defaults via a `newDefaultContext()` wrapper func that pulled from `build.Default` in the same way `newContext()` unconditionally does today.